bodhi/server/views/generic.py

```diff
@@ -67,6 +67,76 @@ def get_latest_updates(request, critpath, security):
     query = query.order_by(models.Update.date_submitted.desc())
     return query.limit(5).all()
 
+def _get_status_counts(basequery, status):
```

Comment:
I should have been more explicit last time - PEP-8 requires exactly two blank lines between top level objects (like functions or classes), so now the test is upset that there is only one line here:

```
/home/vagrant/bodhi/bodhi/server/views/generic.py:70:1: E302 expected 2 blank lines, found 1
```

You can run the tests in vagrant by typing ```btest``` - you should see it say "OK" at the end and not "FAILED". When it fails it'll tell you what happened if you scroll above the coverage report.